lib/services/platform-service.ts

```diff
@@ -256,18 +256,18 @@ export class PlatformService extends EventEmitter implements IPlatformService {
 		return true;
 	}
 
-	public async validateOptions(provision: any, projectData: IProjectData, platform?: string): Promise<boolean> {
+	public async validateOptions(provision: true | string, projectData: IProjectData, platform?: string): Promise<boolean> {
```

Comment:
And if it were string it would still fail and allow us to switch them.
I can't wait for this: https://github.com/Microsoft/TypeScript/issues/202
It will let us better type such strings and limit where they can be used.